server.js

```diff
@@ -47,33 +62,45 @@ app.post("/account/register", (req, res) => {
 
 });
 
-app.post("/account/login", (req, res) => {
-	if (!(req.body.email && req.body.password)) {
-		res.status(StatusCodes.BAD_REQUEST).json({ "error": "Please send an email and password!" });
-		return;
-	}
 
-	const params = [req.body.email];
-	db.get(query.GET_ACCOUNT, params, function(err, row) {
-		if (err) {
-			res.status(StatusCodes.INTERNAL_SERVER_ERROR).json({ "error": err.message });
-			return console.error(err);
+app.post("/account/login", (req, res, next) => {
+	passport.authenticate("local", (err, user, info) => {
+		if (err) throw err;
+		if (!user) res.send("No User Exists");
+		else {
+		  req.logIn(user, (err) => {
+			if (err) throw err;
+			res.send("Successfully Authenticated");
+		  });
 		}
-
-		if (row) {
-			bcrypt.compare(req.body.password, row.password, function(err, result) {
-				if (result) {
-					res.sendStatus(StatusCodes.OK);
-				} else {
-					res.status(StatusCodes.UNAUTHORIZED).json({ "Message": error.INCORRECT_PASSWORD });
-				}
-			});
-		} else {
-			res.status(StatusCodes.NOT_FOUND).json({
-				"message": error.USER_NOT_FOUND
-			});
-		}
-	});
+	})(req, res, next);
+
+	// if (!(req.body.email && req.body.password)) {
```

Comment:
Why is there a big comment here? Use it or lose it...

It's not elegant to keep big comments 😋